### PR TITLE
LPS-120376 Created a merged single sql in portal

### PR DESCRIPTION
**Pull request by dantewang**

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test:relevant

File: benchmarks/build.xml [hunk 1]
```diff
@@ -69,14 +99,52 @@
...
-				includes="*.csv,*.sql,benchmarks-actual.properties,output/**,temp/**"
+				includes="*.csv,*.sql,benchmarks-actual.properties,liferay/**,output/**,temp/**"
 			/>
 		</delete>
 	</target>
+
+	<target depends="clean, build-sample-sql" name="merge-sql">
+		<property file="benchmarks.properties" />
+
+		<concat append="true" destfile="${sample-data-sql}" if:true="${sample.sql.output.merge}">
```
Comment (tinatian) [inline benchmarks/build.xml:115]: @dantewang @slnn This can be an "else" of `<equals arg1="${sample.sql.output.merge}" arg2="false" />`.

**Comment by tinatian** [comment]
@dantewang @slnn Please remove build.properties, move all the content to benchmark.properties. I do not see a point we have two properties files here.

Tina.

**Comment by slnn** [comment]
Resend in https://github.com/dantewang/liferay-portal/pull/1759

**Pull request closed without merging**